Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/ConfirmedStatusCard.kt

```diff
@@ -0,0 +1,77 @@
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.TwoGPlusPCR
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.TwoGPlusRAT
+import de.rki.coronawarnapp.covidcertificate.person.ui.details.PersonDetailsAdapter
+import de.rki.coronawarnapp.covidcertificate.person.ui.overview.PersonColorShade
+import de.rki.coronawarnapp.databinding.ConfirmedStatusCardBinding
+import de.rki.coronawarnapp.util.ContextExtensions.getDrawableCompat
+import de.rki.coronawarnapp.util.lists.diffutil.HasPayloadDiffer
+import setTextWithUrl
+
+class ConfirmedStatusCard(parent: ViewGroup) :
+    PersonDetailsAdapter.PersonDetailsItemVH<ConfirmedStatusCard.Item, ConfirmedStatusCardBinding>(
+        layoutRes = R.layout.confirmed_status_card,
+        parent = parent
+    ) {
+
+    override val viewBinding: Lazy<ConfirmedStatusCardBinding> = lazy {
+        ConfirmedStatusCardBinding.bind(itemView)
+    }
+
+    override val onBindData: ConfirmedStatusCardBinding.(
+        item: Item,
+        payloads: List<Any>
+    ) -> Unit = { item, payloads ->
+
+        when (item.admissionState) {
+            is TwoG -> {
+                subtitle.text = context.resources.getString(R.string.confirmed_status_2g_badge)
+                badge.text = context.resources.getString(R.string.confirmed_status_2g_badge)
+                body.text = context.resources.getString(R.string.confirmed_status_2g_body)
+            }
+            is TwoGPlusPCR -> {
+                subtitle.text = context.resources.getString(R.string.confirmed_status_2g_pcr_subtitle)
+                badge.text = context.resources.getString(R.string.confirmed_status_2g_plus_badge)
+                body.text = context.resources.getString(R.string.confirmed_status_2g_plus_body)
+            }
+            is TwoGPlusRAT -> {
+                subtitle.text = context.resources.getString(R.string.confirmed_status_2g_rat_subtitle)
+                badge.text = context.resources.getString(R.string.confirmed_status_2g_plus_badge)
+                body.text = context.resources.getString(R.string.confirmed_status_2g_plus_body)
+            }
+            is ThreeGWithRAT -> {
+                subtitle.text = context.resources.getString(R.string.confirmed_status_3g_badge)
+                badge.text = context.resources.getString(R.string.confirmed_status_3g_badge)
+                body.text = context.resources.getString(R.string.confirmed_status_3g_body)
+            }
+            is ThreeGWithPCR -> {
+                subtitle.text = context.resources.getString(R.string.confirmed_status_3g_plus_badge)
+                badge.text = context.resources.getString(R.string.confirmed_status_3g_plus_badge)
+                body.text = context.resources.getString(R.string.confirmed_status_3g_plus_body)
+            }
+        }
+
+        badge.background = context.getDrawableCompat(item.colorShade.admissionBadgeBg)
+
+        faq.setTextWithUrl(
```

Comment:
FAQ link should point to `de` when user loacle is `de` and to `en` if it's anything else. I've done something similar here https://github.com/corona-warn-app/cwa-app-android/pull/4260/files